userspace/otpilot/src/spi_device.rs

```diff
@@ -280,4 +289,24 @@ impl SpiDevice for SpiDeviceImpl {
 
         Ok(())
     }
+
+    fn configure_addresses(&self, address_config: AddressConfig) -> TockResult<()> {
+        let mut buf = [0u8; ADDRESS_CONFIG_LEN];
+
+        {
+            // Scope for cursor (which doesn't implement Drop).
+            // We need cursor to go out of scope so that we can use buf further down.
+            let cursor = Cursor::new(&mut buf);
+            if address_config.to_wire(cursor).is_err() {
+                return Err(TockError::Format);
+            }
+        }
+
+        // We want this to go out of scope after executing the command
```

Comment:
I think it's still correct. The point is that we're declaring a variable prefixed with `_` since it's unused. Removing the declaration would have the side effect of dropping the object immediately.

Reply:
OK, I missed that point.  Maybe add to the comment?  Something like:

    // Assign return value to a(n unused) variable to keep it alive while the command is being executed.

Reply:
Done.